File: blog-sever/server-java/src/main/java/com/site/blog/constants/property/SiteProperty.java
package com.site.blog.constants.property;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

/**
 * @author yanni
 * @date time 2022/1/20 2:28
 * @modified By:
 */
@Component
@Data
@ConfigurationProperties(value = "myblog.site")
public class SiteProperty {
    private String ip;
    private String name;
}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/constants/property/SiteProperty.kt
package com.site.blog.constants.property

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * @author yanni
 * @date time 2022/1/20 2:28
 * @modified By:
 */
@Component
@ConfigurationProperties(value = "myblog.site")
data class SiteProperty (
    var ip: String? = null,
    var  name: String? = null
    )